platform/core/src/services/ViewportGridService/ViewportGridService.ts

```diff
@@ -115,12 +115,8 @@ class ViewportGridService extends PubSubService {
     this.serviceImplementation._onModeExit();
   }
 
-  public setCachedLayout({ cacheId, cachedLayout }) {
-    this.serviceImplementation._setCachedLayout({ cacheId, cachedLayout });
-  }
-
-  public restoreCachedLayout(cacheId) {
-    this.serviceImplementation._restoreCachedLayout(cacheId);
+  public restoreCachedLayout(cached) {
```

Comment:
It is a bit unexpected that we still have a `restoreCachedLayout` because this service likely no longer does the caching. Please consider removing this function and simply using `set``.